releng/org.obeonetwork.dsl.sysml.parent/pom.xml

```diff
@@ -40,8 +40,8 @@ Contributors:
 
   <properties>
     <tycho-version>0.19.0</tycho-version>
-    <!-- <umlcore-site>https://s3-eu-west-1.amazonaws.com/obeo-umldesigner-nightly/master/repository</umlcore-site> FRB faire pointer vers l updatesite d'umlCore--> 
-    <umlcore-site>file:///C:/github/UML-Designer/packaging/org.obeonetwork.dsl.uml2.core.update/target/repository</umlcore-site>
+    <umlcore-site>https://s3-eu-west-1.amazonaws.com/obeo-umldesigner-nightly/master/repository</umlcore-site>
```

Comment:
this should be https://s3-eu-west-1.amazonaws.com/obeo-umldesigner-nightly/master/CORE/repository